website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -200,3 +200,7 @@ fatal: reference is not a tree: e845be54e6dc72342d5a8f814c8b3316ee220312
 ```
 
 Double-check that your PR isn't trying to merge using a commit that belongs to a fork of the repository attached to your dbt project.
+
+### CI Job Not Triggering for Virtual Private dbt Customers ###
```

Comment:
changing to sentence case per [style guide](https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md)

```suggestion
### CI job not triggering for Virtual Private dbt Customers ###
```